README.md

```diff
@@ -1,215 +1,126 @@
-# Spring Web MVC Lab 1
-
-Congratulations! You have been hired by Access Camp and for your first task, you
-have been tasked with building out a website to log campers with their
-activities.
-
-Your task is to build the API according to the deliverables below.
-
-## Setup
-
-Use [Spring Initializr](https://start.spring.io/) to create a project with the
-following properties:
-
-- Java 11
-- Maven Build
-
-And add the following dependencies:
-
-- Spring Web.
-- Spring Data JPA.
-- A database of your choice (H2, Postgres, MySQL).
-
-Generate the project and open it up in IntelliJ.
-
-## Models
-
-You need to create the following relationships:
-
-- A `Camper` has many `Signups`, and has many `Activity`s through `Signup`s
```

Comment:
Is the camper project totally gone or will it be moved to a later module?

Reply:
This will turn into the Spring project; so nope not completely gone! It might be modified though to better model the Palantir capstone though slightly.

Reply:
Awesome! I liked the Camper project